tsl/test/sql/cagg_bgw_dist_ht.sql

```diff
@@ -20,6 +20,7 @@ FROM (
 ) a;
 
 GRANT USAGE ON FOREIGN SERVER :DATA_NODE_1, :DATA_NODE_2, :DATA_NODE_3 TO PUBLIC;
+GRANT CREATE ON SCHEMA public TO :ROLE_DEFAULT_PERM_USER;
```

Comment:
These grants are already done in `test/runner.sh` so why do you need to set them here?

Reply:
I think we don't grant permission to these roles on a foreign server in runner.sh.

Reply:
This was implemented in https://github.com/timescale/timescaledb/pull/3712 I think. I don't see why we should do explicit data node handling here.

Reply:
I'm actually suggesting to add a comment so that it is clear, not add this code. Right now it is not clear why this solves any issues.

Reply:
We're doing this in hundreds of files at this point though.

Reply:
That is not a reason to not add a comment here though. Since I got confused about this, I would expect the next developer to be confused as well.

Reply:
The question is, do we add this comment everywhere?

Reply:
If you want to, but in this case it's only these statements that are confusing since they are not adding permissions on the access node, they are adding them on the data nodes, so I think it is good to just clarify these cases.